src/components/Layout/Contact.tsx

```diff
@@ -13,73 +13,145 @@ import {
   Text,
   Grid,
   GridItem,
-  Textarea
+  Textarea,
+  useToast
 } from "@chakra-ui/react";
-
+import { sendEmail } from "../../utils/fetchData";
+import { error } from "console";
```

Comment:
BLOCKING: I think this is an unused import. Check your terminal for unused imports and remove them.

Reply:
removed